### ISSUE TITLE
Wal-g 0.2.9 crashes when deleting old backups

### ISSUE THREAD

**Issue by CyberDem0n** [wal-g/wal-g#319]
```
wal-g delete before FIND_FULL base_0000000B0000B3D300000071_D_0000000B0000B35A0000009A --confirm
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/wal-g/wal-g/cmd.fetchLSN(0x16a4100, 0xc0078d1260, 0xc00009826b, 0x18)
        /home/travis/gopath/src/github.com/wal-g/wal-g/cmd/delete.go:162 +0x7a
github.com/wal-g/wal-g/cmd.postgresLess(0x16a4100, 0xc0078d1260, 0x16a4100, 0xc00034cde0, 0x2c)
        /home/travis/gopath/src/github.com/wal-g/wal-g/cmd/delete.go:158 +0x35
github.com/wal-g/wal-g/cmd.runDeleteBefore.func1(0x16a4100, 0xc0078d1260, 0x2c)
        /home/travis/gopath/src/github.com/wal-g/wal-g/cmd/delete.go:136 +0x47
github.com/wal-g/wal-g/internal/storages/storage.DeleteObjectsWhere(0x16b85c0, 0xc0003a01c0, 0xc0007f3c60, 0x38, 0x2)
        /home/travis/gopath/src/github.com/wal-g/wal-g/internal/storages/storage/folder.go:37 +0x114
github.com/wal-g/wal-g/cmd.runDeleteBefore(0x20c18a0, 0xc0002fd1d0, 0x2, 0x3)
        /home/travis/gopath/src/github.com/wal-g/wal-g/cmd/delete.go:135 +0x1ac
github.com/wal-g/wal-g/vendor/github.com/spf13/cobra.(*Command).execute(0x20c18a0, 0xc0002fd110, 0x3, 0x3, 0x20c18a0, 0xc0002fd110)
        /home/travis/gopath/src/github.com/wal-g/wal-g/vendor/github.com/spf13/cobra/command.go:766 +0x2cc
github.com/wal-g/wal-g/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x20c1b00, 0x154aac5, 0xe, 0xc00028bf28)
        /home/travis/gopath/src/github.com/wal-g/wal-g/vendor/github.com/spf13/cobra/command.go:852 +0x2fd
github.com/wal-g/wal-g/vendor/github.com/spf13/cobra.(*Command).Execute(0x20c1b00, 0x0, 0x0)
        /home/travis/gopath/src/github.com/wal-g/wal-g/vendor/github.com/spf13/cobra/command.go:800 +0x2b
github.com/wal-g/wal-g/cmd.Execute()
        /home/travis/gopath/src/github.com/wal-g/wal-g/cmd/root.go:30 +0x2d
main.main()
        /home/travis/gopath/src/github.com/wal-g/wal-g/main/wal-g/main.go:31 +0x7f
```

**Comment by g0djan** [comment]
Thanks, it's known bug. It is already fixed and will be added to new release.

**Comment by x4m** [comment]
So, let's make a pre-release with fixes?
@g0djan can you please send me a list of known fixes so that I could cut a release?

Also, maybe we should add it to integration tests?

**Comment by nico1106** [comment]
I have the same problem. No backups and WAL-Files are deleted. Is there already a new version out there?

**Comment by x4m** [comment]
@nico1106 yes, there is prerelease fix this issue fixed 0.2.10
